bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -195,6 +198,12 @@
     // Stats
     protected static final String ENABLE_TASK_EXECUTION_STATS = "enableTaskExecutionStats";
 
+    // Allocator configuration
+    protected static final String ALLOCATOR_POOLING_POLICY = "allocatorPoolingPolicy";
```

Comment:
shall we add these settings to AbstractConfiguration? so that client can also configure.

Reply:
@merlimat I have this comment about adding these allocator settings to AbstractConfiguration. so it can be used across client and server. what is your thought?

Reply:
@sijie  I missed it since it was hidden by github. Fixed now